whelk-core/src/main/groovy/whelk/search2/Stats.java

```diff
@@ -123,6 +127,9 @@ private Map<String, Object> buildSliceByDimension(Map<Property, Map<PropertyValu
             var sliceNode = new LinkedHashMap<>();
             var isRange = rangeProps.contains(property);
             var observations = getObservations(buckets, isRange ? queryTree.removeTopLevelPropValueWithRangeIfPropEquals(property) : queryTree, nonQueryParams);
+            if (property.name().equals(Disambiguate.RDF_TYPE)) {
```

Comment:
Use `property.isType()` to catch both work and instance types (`hasInstanceType`/`instanceOfType` in addition to `rdf:type`).